modules/core/shared/src/main/scala/lucuma/core/geom/Shape.scala

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.geom
+
+import lucuma.core.math.Offset
```

Comment:
Shape seems added and deleted. Could instead be git moved?

Reply:
Git didn't pick it up as a move, only for this file, probably because it's a small file and all the scaladoc was reformatted, going over the similarity threshold. I agree this is far from ideal since it breaks continuity.

We *could* split the commit in two, one with the move and another one with the modification and then git will probably pick it up.